src/test/java/io/github/sashirestela/cleverclient/CleverClientTest.java

```diff
@@ -19,18 +19,31 @@ class CleverClientTest {
     @Test
     void shouldSetPropertiesToDefaultValuesWhenBuilderIsCalledWithoutThoseProperties() {
         var cleverClient = CleverClient.builder()
-                .urlBase("https://test")
+                .baseUrl("https://test")
                 .build();
         assertEquals(List.of(), cleverClient.getHeaders());
         assertEquals(HttpClient.Version.HTTP_2, cleverClient.getHttpClient().version());
-        assertNotNull(cleverClient.getUrlBase());
+        assertNotNull(cleverClient.getBaseUrl());
+        assertNotNull(cleverClient.getHttpProcessor());
+    }
+
+    @Test
+    void shouldSetPropertiesToDefaultValuesWhenBuilderIsCalledWithoutThoseProperties2() {
```

Comment:
Instead of the suffix "2" try to use a meaningful name describing the scenario that you are testing, please.